WebCalc/WebCalc/Pages/Index.cshtml.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using WebCalc.Models;
+using WebCalc.Operators;
+
+namespace WebCalc.Pages;
+
+[BindProperties]
+public class Main : PageModel
```

Comment:
```suggestion
public class CalculatorPageModel : PageModel
```
Ну не Main же. (-0.5)